core-rust/core-api-server/src/core_api/handlers/lts/state_account_all_fungible_resource_balances.rs

```diff
@@ -0,0 +1,84 @@
+use crate::core_api::*;
+use radix_engine::types::{Decimal, IndexMap};
+use state_manager::store::traits::QueryableProofStore;
+use state_manager::{
+    jni::state_manager::ActualStateManager,
+    query::{dump_component_state, VaultData},
+};
+
+#[tracing::instrument(skip(state), err(Debug))]
+pub(crate) async fn handle_rc_state_account_all_fungible_resource_balances(
```

Comment:
Should probably have `_lts_` in the name now (and same for all methods in these files)